cmd/otelcontribcol/components.go

```diff
@@ -40,6 +40,7 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourcedetectionprocessor"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/carbonreceiver"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/collectdreceiver"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/ecstaskmetadataendpointreceiver"
```

Comment:
Let's prefix all the AWS modules with `aws`. How about just `awsecsmetadatareceiver`?

Reply:
It's a good idea.